data/mods/Aftershock/mutations/mutations.json

```diff
@@ -8,5 +8,292 @@
     "description": "NPC trait that makes monsters see it as a Prepnet.  It is a bug if you have it.",
     "player_display": false,
     "threshold": true
+  },
+  {
+    "type": "mutation_category",
+    "id": "MIGO",
+    "name": "YUGGOTH",
+    "threshold_mut": "THRESH_YUGGOTH",
+    "mutagen_message": "Strange memories of another world overwhelm you…",
+    "iv_message": "You look into your own cells and change them.",
+    "iv_pain": 10,
+    "iv_morale": 25,
+    "iv_morale_max": 100,
+    "iv_sleep": true,
+    "//": "Should be out for two minutes.  Nightmare realm becomes home",
+    "iv_sleep_dur": 20,
+    "iv_sleep_message": "You gaze inside, past your fear, past good and evil, into the future.",
+    "memorial_message": "Prepared the way into the galaxy."
+  },
+  {
+    "type": "mutation",
+    "id": "MIGO_HEAT_RESIST",
+    "name": "Mi-go acclimatization",
+    "points": 4,
+    "bodytemp_modifiers": [ -5500, -5500 ],
+    "description": "Your body is capable of living more comfortably in the mi-go atmosphere.  Your scalp develops fronds that act as heat sinks.",
+    "player_display": true,
+    "threshreq": [ "THRESH_YUGGOTH" ],
+    "starting_trait": false,
+    "types": [ "METABOLISM" ],
+    "active": true,
+    "category": [ "MIGO" ],
+    "prereqs": [ "WARM_NATURED" ]
+  },
+  {
+    "type": "mutation",
+    "id": "MOUTH_TENTACLES",
+    "name": "Mouth Tentacles",
+    "points": 1,
+    "visibility": 8,
+    "ugliness": 5,
+    "description": "A set of tentacles surrounds your mouth.  They allow you to eat twice as fast.  Slightly decreases wet penalties.",
+    "prereqs": [ "MOUTH_FLAPS" ],
+    "cancels": [ "MANDIBLES" ],
+    "category": [ "CEPHALOPOD", "MIGO" ],
+    "wet_protection": [ { "part": "MOUTH", "neutral": 4 } ]
+  },
+  {
+    "type": "mutation",
+    "id": "MOUTH_FLAPS",
+    "name": "Mouth Flaps",
+    "points": -1,
+    "visibility": 7,
+    "ugliness": 6,
+    "description": "Skin tabs and odd flaps of skin surround your mouth.  They don't affect your eating, but are unpleasant to look at.",
+    "category": [ "CHIMERA", "MIGO" ],
+    "leads_to": [ "MOUTH_TENTACLES", "MANDIBLES" ]
+  },
+  {
+    "type": "mutation",
+    "id": "WARM_NATURED",
+    "name": "Warm Natured",
+    "points": 2,
+    "description": "Your body becomes much more efficient at distributing heat from itself.",
+    "category": [ "MIGO" ],
+    "leads_to": [ "MIGO_HEAT_RESIST" ],
+    "bodytemp_modifiers": [ -1000, -1000 ]
+  },
+  {
+    "type": "mutation",
+    "id": "INT_UP",
```

Comment:
I'd advice against overwritting vanilla mutaions in aftershock.
It tends to result in inter-mod compatibility issues and for my limited experience it also tends to require a lot of maintenance. The former is a problem because it seems that most players who use aftershock also tend to enable every single mod in the launcher, and the later is just annoying.

My suggestion is that you make all these overwrites into mi-go exclusive traits.

Reply:
That makes sense.  I'll basically leave them the same except for ID, description and then change the categories to only MIGO?

Reply:
Yeah just that pretty much.

Reply:
Do PRED1 and PRED2 tie back to C++?  Because I don't see what in the JSON would allow increased experience for combat skills. :(

Reply:
Yeah their effects are hardcoded by IDs.
It shouldnt be that hard to change them to work through flags. If you want an example, you can search how cannibal and psycopath currently work through flags.

If you give me till next weekend I can also do it for you.

Reply:
I can absolutely wait til you have time. :) Thank you!